CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## master (unreleased)
 
+* [#83](https://github.com/clojure-emacs/orchard/pull/83): Ignore non file URLs when checking for directory or file extensions
```

Comment:
You should also put "Bugs fixed" above this entry.

Reply:
Oh yeah, is see. Sorry